### DISCUSSION TITLE
Recorded RT greater than trial_duration

### DISCUSSION THREAD

**Discussion by marton-balazs-kovacs** [jspsych/jsPsych#2692] [Q&A]
Hi All,

I have one modified Stroop and one prime probe task where I use a calibration phase to calculate a deadline for responses during the test phase. The deadline is added to the test phase in the `on_start` event.

I have collected a lot of data and in most of the cases, the RT is smaller than the trial duration. However, in a few instances, the RT is greater than the trial_duration. In the most extreme case with more than 1 second.

**Do you have any tips on how is this possible?** 

When I look at these strange instances I do not see any particular pattern.

I tried to reproduce this result using my computer and running it on localhost (I used jatos for data collection), but I could not get the rt higher than the trial_duration. It would be important to know the source of the mistakes since a great proportion of the trials during the test phase were timeouts.

Here is the code that I used:
[](https://github.com/marton-balazs-kovacs/deadline-cse/blob/main/src/experiment.js)

Best,
Marton

**Comment by jodeleeuw** [comment]
Hi @marton-balazs-kovacs 

Are you using a custom plugin to run any of these tasks?

**Comment by marton-balazs-kovacs** [comment]
Hi @jodeleeuw,

I was using JATOS to collect the data.
